README.md

```
@@ -4,158 +4,73 @@
-**Option 2: Config file**
-
-Add to `~/.openclaw/openclaw.json`:
-
-```json
-{
-  "agent": {
-    "workspace": "/path/to/custom/workspace"
-  }
-}
-```
+## Configuration
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)